### PR TITLE
feat: added query to cardano-node for node version

### PR DESCRIPTION
**Pull request by t-dallas**
# Description

This PR adds query to cardano-node for node version

# Testing

`yarn test`

**Remember to add to .env file and .env.test CARDANO_NODE_PATH to its corresponding cardano-node path**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
-
   tests:
-
     runs-on: ubuntu-latest
```
Comment (AlanVerbner) [inline .github/workflows/ci.yml:11]: are this changes needed?

**Review by AlanVerbner — DISMISSED** [review]
looks great! Let's wait until #148 is merged

File: .github/workflows/ci.yml (hunk 1 above)
Comment (rhyslbw) [reply .github/workflows/ci.yml:11]: Not required, but may as well tidy up

**Review by rhyslbw — APPROVED** [review]
@AlanVerbner approved the state, but is stale due to rebase https://github.com/input-output-hk/cardano-rosetta/pull/151#pullrequestreview-477130385
